### PR TITLE
T978: Support PowerDNS Recursor outbound queries over IPv6.

### PR DESCRIPTION
**Pull request by gsadams**
This requires adding a query-local-address6 setting to enable outbound
IPv6 queries in general, and also formatting upstream nameserver IPv6
addresses in such a way that Recursor can parse them.

### CODE REVIEW

**Comment by c-po** [comment]
Thx for the contribution. Usually we prefix our commit headline with the phabricator task id, thus commits can be automassigned to tasks. Can you please reword your commit?

File: src/conf_mode/dns_forwarding.py [hunk 1]
```diff
@@ -39,6 +39,7 @@
 allow-from=0.0.0.0/0, ::/0
 log-common-errors=yes
 non-local-bind=yes
+query-local-address6=::
```
Comment (c-po) [inline src/conf_mode/dns_forwarding.py:43]: Could you please add `query-local-address=0.0.0.0/0` for completion, too? 
Then we have both IPv4 and IPv6 statements.

**Comment by gsadams** [comment]
Done.

**Review by c-po — CHANGES_REQUESTED** [review]
Could you please add query-local-address=0.0.0.0/0 for completion, too?
Then we have both IPv4 and IPv6 statements.

File: src/conf_mode/dns_forwarding.py (hunk 1 above)
Comment (gsadams) [reply src/conf_mode/dns_forwarding.py:43]: Done (but without the /0; it doesn't like that). This is the default value, so I didn't include it, but it does seem valuable to be explicit.

**Review by c-po — APPROVED** [review]

**Pull request merged**